.pre-commit-config.yaml

```diff
@@ -1,15 +1,26 @@
 ---
 repos:
-# isort for jupyter notebooks
-  - repo: https://github.com/nbQA-dev/nbQA
-    rev: 1.3.1
+
```

Comment:
This file could use some gardening.

(e.g., remove extra new lines, decide what to do with commented code)

Reply:
Cleaned up this pre-commit file a bit